sdk/monitor/Azure.Monitor.OpenTelemetry/README.md

```diff
@@ -1,19 +1,101 @@
+
+#### Example 2
+
+To enable Azure Monitor Distro with a hard-coded connection string, add `AddAzureMonitor()` to your `Program.cs` with the `AzureMonitorOptions` containing the connection string.
+
+```C#
+// This method gets called by the runtime. Use this method to add services to the container.
+var builder = WebApplication.CreateBuilder(args);
+
+// The following line enables Azure Monitor Distro with hard-coded connection string.
+builder.Services.AddAzureMonitor(o => o.ConnectionString = "InstrumentationKey=00000000-0000-0000-0000-000000000000");
+
+// This code adds other services for your application.
+builder.Services.AddMvc();
+
+var app = builder.Build();
+```
+
+Note that in the examples above, `AddAzureMonitor` is added to the `IServiceCollection` in the `Program.cs` file. You can also add it in the `ConfigureServices` method of your `Startup.cs` file.
+
 ### Authenticate the client
 
+Azure Active Directory (AAD) authentication is an optional feature that can be used with Azure Monitor Distro. To enable AAD authentication, set the `Credential` property in `AzureMonitorOptions`. This is made easy with the [Azure Identity library][identity], which provides support for authenticating Azure SDK clients with their corresponding Azure services.
+
+```C#
+// Call AddAzureMonitor and set Credential to authenticate through Active Directory.
+builder.Services.AddAzureMonitor(o =>
+{
+    o.ConnectionString = "InstrumentationKey=00000000-0000-0000-0000-000000000000";
+    o.Credential = new DefaultAzureCredential();
+});
+```
+
+With this configuration, the Azure Monitor Distro will use the credentials of the currently logged-in user or of the service principal to authenticate and send telemetry data to Azure Monitor.
+
+Note that the `Credential` property is optional. If it is not set, Azure Monitor Distro will use the Instrumentation Key from the Connection String to send data to Azure Monitor.
+
 ## Key concepts
 
+The Azure Monitor Distro uses instrumentation libraries from the .NET OpenTelemetry SDK to provide a convenient and easy way to export telemetry data to Azure Monitor.
+
 ## Examples
 
+Refer to [`Program.cs`](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.OpenTelemetry/tests/Azure.Monitor.OpenTelemetry.Demo/Program.cs) for a complete demo.
+
 ## Troubleshooting
 
+The Azure Monitor Distro uses EventSource for its own internal logging. The logs are available to any EventListener by opting into the source named "OpenTelemetry-AzureMonitor-Exporter".
+
+OpenTelemetry also provides it's own [self-diagnostics feature](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/README.md#troubleshooting) to collect internal logs.
+An example of this is available in our demo project [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Demo/OTEL_DIAGNOSTICS.json).
+
+
 ## Next steps
```

Comment:
I think the natural next steps would be stuff like
1. Go to Azure Portal and check out app insights
2. Great a custom dashboard or set an alert
3. File a support ticket if you're having issues